lucet-runtime/lucet-runtime-internals/src/region.rs

```diff
@@ -67,6 +68,7 @@ pub trait RegionInternal: Send + Sync {
     fn expand_heap(&self, slot: &Slot, start: u32, len: u32) -> Result<(), Error>;
 
     fn reset_heap(&self, alloc: &mut Alloc, module: &dyn Module) -> Result<(), Error>;
+    fn heap_memory_size_limit(&self) -> usize;
```

Comment:
I would recommend making this instead an accessor for the `Limits`, just for extra flexibility